### ISSUE TITLE
Mark footer Slack CTA emoji as decorative with aria-hidden="true"

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/areweheadlessyet#47]
This will improve the accessibility of the Slack CTA.

---

Assigning to @shyusu4 as part of her Outreachy project.

**Comment by thibaudcolas** [comment]
Fixed in #49.
